x-pack/plugins/actions/server/usage/actions_usage_collector.ts

```diff
@@ -4,18 +4,39 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { UsageCollectionSetup } from 'src/plugins/usage_collection/server';
+import { MakeSchemaFrom, UsageCollectionSetup } from 'src/plugins/usage_collection/server';
 import { get } from 'lodash';
 import { TaskManagerStartContract } from '../../../task_manager/server';
 import { ActionsUsage } from './types';
 
+const byTypeSchema: MakeSchemaFrom<ActionsUsage>['count_by_type'] = {
```

Comment:
@mindbat did you want the alerting team to keep updating this list on each release when new actions exist? or will it be possible with the `DYNAMIC_KEY` to automatically handle new types of actions per release on your end?

Reply:
I'm a little fuzzy on the code here (not a ts dev 😅 ), but it's my understanding that @afharo's `DYNAMIC_KEY` in the schema that gets handed off to my team will be enough for us to know "this key's value will have this type, but it's name might change"